Library/Homebrew/utils/gems.rb

```diff
@@ -12,6 +12,9 @@ module Homebrew
   # After updating this, run `brew vendor-gems --update=--bundler`.
   HOMEBREW_BUNDLER_VERSION = "2.4.18"
 
+  GEMGROUPS_FILE = "#{HOMEBREW_LIBRARY_PATH}/vendor/bundle/ruby/.homebrew_gemgroups"
```

Comment:
How about making this a `Pathname`?

Reply:
Yes, just realised `HOMEBREW_LIBRARY_PATH` itself is a Pathname so this should be safe to do.